File: Release/WindowsPhone7/PhoneToolkitSample/Data/MoviesInCategory.cs
// (c) Copyright Microsoft Corporation.
// This source is subject to the Microsoft Public License (Ms-PL).
// Please see http://go.microsoft.com/fwlink/?LinkID=131993 for details.
// All other rights reserved.

using System.Collections.ObjectModel;

namespace PhoneToolkitSample.Data
{
    public class MoviesInCategory : ObservableCollection<Movie>
    {
        public MoviesInCategory(string category)
        {
            Key = category;
        }

        public string Key { get; set; }

        public string GetMore { get { return "More " + Key; } }
    }
}

File: Release/WindowsPhone71/PhoneToolkitSample/Data/MoviesInCategory.vb
' (c) Copyright Microsoft Corporation.
' This source is subject to the Microsoft Public License (Ms-PL).
' Please see http://go.microsoft.com/fwlink/?LinkID=131993 for details.
' All other rights reserved.

Imports System.Collections.ObjectModel

Namespace Data

    Public Class MoviesInCategory
        Inherits ObservableCollection(Of Movie)
        Public Sub New(ByVal category As String)
            Key = category
        End Sub

        Public Property Key As String

        Public ReadOnly Property GetMore As String
            Get
                Return "More " & Key
            End Get
        End Property
    End Class

End Namespace
